packages/finoaconnect/src/index.ts

```diff
@@ -0,0 +1,78 @@
+import { FinoaEIP1193Provider } from '@finoa/finoa-connect-sdk'
+import type { Chain, WalletInit } from '@web3-onboard/common'
+
+export interface FinoaWalletOption {
```

Comment:
Adding this type to the docs and readme would be helpful for devs

Reply:
Also adding notes on what the props are would be helpful

Reply:
done. Any thoughts?